templates/ts-apollo-mongodb-runtime-backend/.env

```diff
@@ -0,0 +1,6 @@
+DB_USER=mongodb
```

Comment:
This is from datasync example right? We use there some dodgy Red Hat Mongo for compatibility. 
Normal Mongo do not require any authentication etc. We might just not need it for upstream context

Reply:
No this is the exact config from `graphback config`

Reply:
Do we create .env file there?

Reply:
no I coped the database values from the docker-compose file